misc/starfighter.appdata.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>starfighter.desktop</id>
+  <metadata_license>BY-SA-4.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <name>Project: Starfighte</name>
+  <summary>Space shoot 'em up game.</summary>
+  <description>
+    <p>
+      Project: Starfighter is a space shoot 'em up game originally developed
+by Parallel Realities in 2002, and released in 2003. You assume the role
+of Chris Bainfield in his quest to put an end to WEAPCO, the weapons
+corporation which has enslaved the galaxy.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot>
+      <image>https://pr-starfighter.github.io/images/screenshot1.png</image>
+      <caption>Explosions</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://pr-starfighter.github.io/images/screenshot2.png</image>
+      <caption>Main Menu</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://pr-starfighter.github.io/images/screenshot3.png</image>
+      <caption>Boss Fight</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://pr-starfighter.github.io/images/screenshot4.png</image>
+      <caption>Mines and Asteroids</caption>
+    </screenshot>
+  </screenshots>
+  <url type="homepage">https://pr-starfighter.github.io/</url>
+  <update_contact>diligentcircle@riseup.net</update_contact>
+  <content_rating type="oars-1.0">
+    <content_attribute id="violence-cartoon">none</content_attribute>
+    <content_attribute id="violence-fantasy">none</content_attribute>
+    <content_attribute id="violence-realistic">moderate</content_attribute>
+    <content_attribute id="violence-bloodshed">none</content_attribute>
+    <content_attribute id="violence-sexual">none</content_attribute>
+    <content_attribute id="drugs-alcohol">none</content_attribute>
+    <content_attribute id="drugs-narcotics">none</content_attribute>
+    <content_attribute id="drugs-tobacco">none</content_attribute>
+    <content_attribute id="sex-nudity">none</content_attribute>
+    <content_attribute id="sex-themes">none</content_attribute>
+    <content_attribute id="language-profanity">none</content_attribute>
```

Comment:
I believe this should be "moderate", so I'll be changing it to that.

🕵️

Reply:
Do it. I have to admit that I eyeballed those.

Reply:
Already done. :slightly_smiling_face: I did it right after I said that. Also found some other improvements to make.

🕵️